src/js/index.js

```diff
@@ -1,4 +1,42 @@
 import renderPage from './page.js';
 import '../css/style.css';
+import API from './modules/api.js';
+import Game from './modules/model/game.js';
+import GameUI from './modules/gameui.js';
 
-renderPage();
\ No newline at end of file
+const nGame = new Game();
+
+renderPage();
+const refreshList = () => {
+  const scoreList = nGame.getAllScore();
+  GameUI.displayScore(scoreList);
+};
+document.querySelector('.score-submit-btn').addEventListener('click', (e) => {
+  e.preventDefault();
+  const fields = e.target.parentElement.getElementsByTagName('input');
+  const name = fields[0].value;
+  const score = fields[1].value;
+  const game = new Game(name, score);
+  game.save();
+  GameUI.addScoreToUI(game, 'list');
+  GameUI.clearFom('game-form');
+});
```

Comment:
- [ ] I would recommend you to add validations to your form, so that empty inputs would not hit your `post api endpoint`. At this moment, this is super possible which is making your code not too efficient. Kindly look into that and fix.
- [ ] I think it is also paramount to ensure name input can only accept string input with strict comparison. At this moment, I can input in numbers and it sees it as strings.

[your-page.webm](https://user-images.githubusercontent.com/26597432/203973712-2681716c-33a7-4562-a603-ceba21a91d2a.webm)